### PR TITLE
surface std error messages to user in real time

### PR DESCRIPTION
**Pull request by MDunitz**
## Need
- Stream task's standard error messages into the task's logger
- Take care (i) not to output partial lines, and (ii) to output lines that were partial on the last iteration.

## Approach
- use pygtail library to read from log file
- log at INFO level

## Tests
- [x] unit tests

## Questions
- is `error_file = [x for x in list(volumes) if "stderr.txt" in x][0]` the best way to get the stderr filename?

### CODE REVIEW

File: WDL/runtime/task.py [hunk 1]
```diff
@@ -229,7 +231,8 @@ def _run(self, logger: logging.Logger, command: str) -> int:
         try:
             container = None
             exit_info = None
-
+            error_file = [x for x in list(volumes) if "stderr.txt" in x][0]
```
Comment (mlin) [inline WDL/runtime/task.py]: I think this should work:

```suggestion
            error_file = os.path.join(self.host_dir, "stderr.txt")
```

File: tests/test_4taskrun.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import signal
 import time
 from .context import WDL
+from testfixtures import log_capture
```
Comment (mlin) [inline tests/test_4taskrun.py]: look like we need testfixtures in requirements.dev.txt

Comment (MDunitz) [reply tests/test_4taskrun.py]: 👍

**Comment by mlin** [comment]
@MDunitz thanks! in the most recent test failure, I think pyre doesn't like the object `pygtail` having the same name as the imported module, `pygtail` -- can you try renaming it?

File: WDL/runtime/task.py [hunk 3]
```diff
@@ -248,13 +251,13 @@ def _run(self, logger: logging.Logger, command: str) -> int:
                 logger.debug(
                     "docker container name = {}, id = {}".format(container.name, container.id)
                 )
-
                 # long-poll for container exit
                 while exit_info is None:
                     try:
+                        for line in pygtail:
+                            logger.info(f"StdError: {line}")
```
Comment (mlin) [inline WDL/runtime/task.py]: we can log `line.rstrip()` here and below, since logger itself adds a newline to the message you give it

**Pull request merged**